frontend/package.json

```
@@ -65,6 +65,7 @@
     "deploy-azure-dev": "REACT_APP_OKTA_URL=https://dev.simplereport.gov REACT_APP_APPINSIGHTS_KEY=29a75c50-8485-4d80-8e1b-41b1a091abb6 REACT_APP_BACKEND_URL=https://dev.simplereport.gov/api/graphql REACT_APP_BASE_URL=https://dev.simplereport.gov REACT_APP_PATIENT_EXPERIENCE_ENABLED=true PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdevapp --destination-path '/app'",
     "deploy-azure-test": "REACT_APP_OKTA_URL=https://test.simplereport.gov REACT_APP_BACKEND_URL=https://test.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttestapp --destination-path '/app'",
     "deploy-azure-pentest": "REACT_APP_OKTA_URL=https://pentest.simplereport.gov REACT_APP_BACKEND_URL=https://pentest.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportpentestapp --destination-path '/app'",
+    "deploy-azure-training": "REACT_APP_OKTA_URL=https://training.simplereport.gov REACT_APP_BACKEND_URL=https://training.simplereport.gov/api/graphql PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttrainingapp --destination-path '/app'",
```

Added the env variable to `main`
https://github.com/CDCgov/prime-simplereport/blob/b51d0d6c8164441384e485a856c6c31d9094e140/frontend/.env#L8
```suggestion
    "deploy-azure-training": "REACT_APP_OKTA_URL=https://training.simplereport.gov REACT_APP_BACKEND_URL=https://training.simplereport.gov/api/graphql REACT_APP_IS_TRAINING_SITE=true PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttrainingapp --destination-path '/app'",
```

---

perfect thanks!